docs/reference/logging/index.md

```diff
@@ -25,6 +25,16 @@ container's logging driver. The following options are supported:
 
 The `docker logs`command is available only for the `json-file` logging driver.  
 
+### The json-file options
+
+The following logging options are supported for the `json-file` logging driver:
+
+   --log-opt max-size=[0-9+][k/m/g/K/M/G]
+   --log-opt max-file=[0-9+]
+
+`max-size` sets the size of each of the log files that are rolled over
+`max-file` sets the maximum number of log files that are rolled over amongst
```

Comment:
@moxiegirl I am trying to denote that one or more digits should be present before the unit field `[k | m | g]`. 

The default value for both these options are empty string. How about something like this?

### The json-file options

The following logging options are supported for the `json-file` logging driver:
```
   --log-opt max-size=""
   --log-opt max-file=""
```
`max-size` Logs that reach this size are rolled over. You can set the size in kilobytes(k), megabytes(m), or gigabytes(g). eg `--log-opt max-size=50m`
`max-file` Maximum number of files that a log is rolled over before being discarded. eg `--log-opt max-file=100`